js/app.js

```diff
@@ -121,8 +118,47 @@ function renderProducts() {
 
 renderProducts();
 
-function handleResultsList() { }
+function handleResultsList() {
```

Comment:
Edit this function to display the amount of votes AND how many times it was viewed

Ex.
function handleResultsList() {
  clear();
  let ul = document.getElementById('product-clicks-list');

  for (let i = 0; i < allProducts.length; i++) {
    let currentProduct = allProducts[i];
    let li = document.createElement('li');
    // Create the sentence with the product's name, clicks, and views
    li.textContent = `Example: ${currentProduct.productName} had ${currentProduct.clicks} votes, and was seen ${currentProduct.views} times.`;
    ul.appendChild(li);
  }
}